addons/point_of_sale/models/pos_session.py

```diff
@@ -398,7 +398,7 @@ def create(self, vals_list):
 
     def unlink(self):
         self.statement_line_ids.unlink()
-        (self.order_seq_id | self.login_number_seq_id).unlink()
+        (self.order_seq_id | self.login_number_seq_id).with_context(force_unlink=True).unlink()
         return super(PosSession, self).unlink()
```

Comment:
```suggestion
        result = super(PosSession, self).unlink()
        (self.order_seq_id | self.login_number_seq_id).unlink()
        return result
```

I think you need to switch the deletion so that no need to `force_unlink`.